### PR TITLE
Typings cleanup

### PR DESCRIPTION
**Pull request by alexeagle**
We had our own fork due to some custom typings, but these have been pushed upstream.

### CODE REVIEW

**Comment by alexeagle** [comment]
Something is fishy. In angular's fork we have  "zone" (https://github.com/angular/DefinitelyTyped/blob/master/zone/zone.d.ts). The one on DefinitelyTyped has "zone.js" (https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/zone.js/zone.js.d.ts#L6)

This PR should have failed on Travis because the zone typings are missing.
the log has under npm install:
```
>> written 7 files:
    - angular-protractor/angular-protractor.d.ts
    - es6-shim/es6-shim.d.ts
    - hammerjs/hammerjs.d.ts
    - jasmine/jasmine.d.ts
    - node/node.d.ts
    - selenium-webdriver/selenium-webdriver.d.ts
    - zone/zone.d.ts
```

Where did it find `zone.d.ts`?? Did it come from the Travis cache even though this PR should bust the cache? Fishy.

**Comment by alexeagle** [comment]
Hmm, I think it's because we give a SHA, and that can be resolved against a repo that doesn't have that in the tree anywhere: this works
https://github.com/DefinitelyTyped/DefinitelyTyped/commit/31e7317c9a0793857109236ef7c7f223305a8aa9
even though that change only appeared on the angular fork. So maybe it's legit that Travis passed.
I'll update the PR though'

**Comment by alexeagle** [comment]
cc @vikerman

File: modules/angular1_router/tsd.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
   "version": "v4",
-  "repo": "angular/DefinitelyTyped",
+  "repo": "DefinitelyTyped/DefinitelyTyped",
```
Comment (rkirov) [inline modules/angular1_router/tsd.json]: no more forks!

File: modules/angular2/manual_typings/globals.d.ts [hunk 2]
```diff
@@ -1,7 +1,52 @@
 /**
- * Declarations angular depends on for compilation to ES6.
- * This file is also used to propagate our transitive typings
- * to users.
+ * Subset of es6-shim typings.
+ * Angular should not require use of ES6 runtime but some API usages are already present.
+ * See https://github.com/angular/angular/issues/5242
+ * TODO(alexeagle): remove methods below which may not be present in targeted browser
```
Comment (rkirov) [inline modules/angular2/manual_typings/globals.d.ts]: Not sure what you are trying to say with that TODO. We are targetting IE9 and it doesn't have any of those.

File: modules/playground/e2e_test/material/progress_linear_spec.ts [hunk 3]
```diff
@@ -14,10 +14,10 @@ describe('md-progress-linear', function() {
     var initialValue = progressBar.getAttribute('aria-valuenow');
 
     incrementButton.click();
-    expect(progressBar.getAttribute('aria-valuenow')).toBeGreaterThan(initialValue);
+    expect(progressBar.getAttribute('aria-valuenow')).toBeGreaterThan(<any>initialValue);
```
Comment (rkirov) [inline modules/playground/e2e_test/material/progress_linear_spec.ts]: what happened here.

**Comment by rkirov** [comment]
looks good, make lint pass

File: modules/angular2/manual_typings/globals.d.ts (hunk 2 above)
Comment (alexeagle) [inline modules/angular2/manual_typings/globals.d.ts]: so maybe the TODO is remove all of these - but I don't imagine we will really do that?

Comment (alexeagle) [inline modules/angular2/manual_typings/globals.d.ts]: @mhevery FYI

File: modules/playground/e2e_test/material/progress_linear_spec.ts (hunk 3 above)
Comment (alexeagle) [inline modules/playground/e2e_test/material/progress_linear_spec.ts]: Turns out this was not type-checked before, our fork of DT did not have the right propagation of the type to the argument of these methods like toBeGreaterThan.

**Comment by btford** [comment]
Is this ready to merge?

**Comment by alexeagle** [comment]
not ready to merge yet

**Comment by alexeagle** [comment]
Reviewed in person with @mhevery

**Comment by alexeagle** [comment]
cc @blakeembrey who helped with the design for this - thank you!

**Pull request closed without merging**